app/export.go

```diff
@@ -55,7 +55,8 @@ func (app *ExocoreApp) ExportAppStateAndValidators(
 ) (servertypes.ExportedApp, error) {
 	// Creates context with current height and checks txs for ctx to be usable by start of next
 	// block
-	ctx := app.NewContext(true, tmproto.Header{Height: app.LastBlockHeight()})
+	ctx := app.NewContext(true, tmproto.Header{Height: app.LastBlockHeight()}).
+		WithChainID(app.ChainID())
```

Comment:
Without this edit, the `export` command complains of `no such key in the store`. The function `app.ChainID()` is available in [`cosmos-sdk v0.47.5-evmos.2`](https://github.com/evmos/cosmos-sdk/releases/tag/v0.47.5-evmos.2), which has been added to `go.mod`. The only change over the previous version is the addition of this method.